localserver.js

```diff
@@ -13,7 +13,8 @@ var middlewareLayers = [express.static(fileSystem.resolve('src')),
                         require('./server_modules/cors').middleware,
                         require('./server_modules/cache').middleware,
                         require('./server_modules/body_builder').middleware,
-                        require('./server_modules/log').middleware];
+                        require('./server_modules/log').middleware,
+                        require('./server_modules/address').middleware];
```

Comment:
Better to place the closing bracket on its own line, to prevent unnecessary changes to the last line (e.g. how the `log` line is now changed).